optaweb-vehicle-routing-backend/pom.xml

```diff
@@ -226,9 +207,9 @@
               </dependency>
             </dependencies>
             <!--
-              kie-parent contains Pitest configuration that doesn't work well for this project and it's not possible
-              to change it without a complete override (combine.self="override"), for example excludedClasses,
-              timeoutConstant.
+              optaplanner-build-parent contains Pitest configuration that doesn't work well for this project
```

Comment:
I wonder how much we use that Pitest configuration in optaplanner-build-parent. But even if we decide to remove it, it's out of scope of this PR.